packages/backend/.env.example

```
@@ -26,8 +26,17 @@
-REDIS_URL=redis://localhost:6379
+# ==== Cache Configuration ====
+# TTL in Sekunden (3600 = 1 Stunde default)
+CACHE_TTL_SECONDS=3600
+# Maximale Anzahl von Cache-Einträgen
+CACHE_MAX_ITEMS=1000
+
+# ==== Rate Limiter Configuration ====
+# Default rate limit window in milliseconds (60000 = 1 minute)
+RATE_LIMITER_WINDOW_MS=60000
+# Default maximum requests per window
+RATE_LIMITER_MAX_REQUESTS=100
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The RATE_LIMITER_MAX_REQUESTS key should go before the RATE_LIMITER_WINDOW_MS key

(UnorderedKey)